frontend/src/components/Utils/time.js

```diff
@@ -18,40 +20,56 @@ export function timestampToTimeFormatted(msTimestamp, timezone = 'America/New_Yo
 /**
  * Format a timestamp (in milliseconds) into a pretty string with just the date.
  *
- * @param {int} msTimestamp 
- * @param {string} timezone 
+ * @param {int} msTimestamp
+ * @param {string} timezone
  * @returns {string} Time formatted into a string like 'Monday, January 19, 1970'.
  */
 export function timestampToDateFormatted(msTimestamp, timezone='America/New_York') {
   const date = new Date(msTimestamp);
-  const formatOptions = { 
-    weekday: 'long', 
-    year: 'numeric', 
-    month: 'long', 
-    day: 'numeric',  
+  const formatOptions = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
     timeZone: timezone
   };
   return date.toLocaleDateString('en-US', formatOptions);
 }
 
-/** 
+/**
  * Format a timestamp (in milliseconds) into a pretty string.
- * 
- * @param {int} msTimestamp 
- * @param {string} timezone 
- * @returns {string} Time formatted into a string like 
+ *
+ * @param {int} msTimestamp
+ * @param {string} timezone
+ * @returns {string} Time formatted into a string like
  * "Monday, January 19, 1970, 02:48 AM"
  */
 export function timestampToFormatted(msTimestamp, timezone = "America/New_York") {
   let date = new Date(msTimestamp);
-  let formatOptions = { 
-    weekday: 'long', 
-    year: 'numeric', 
-    month: 'long', 
-    day: 'numeric',  
-    hour: 'numeric', 
-    minute: '2-digit', 
+  let formatOptions = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: '2-digit',
     timeZone: timezone
```

Comment:
These changes were automatically done by my text editor and I am don't know if its work going into vim and manually changing it back.